src/components/body/pillars/Pillars.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import ReactCSSTransitionReplace from 'react-css-transition-replace';
+import '../../../App.css';
+
+import PageSelector from './PageSelector'
+import Pair from './Pair';
+import Grow from './Grow';
+import Learn from './Learn';
+
+
+class Pillars extends Component {
+  constructor() {
+    super();
+    this.state = {
+      activePillar: 'pair-button'
+    };
+    this.handleSelection = this.handleSelection.bind(this);
+  }
+
+  renderContent() {
+    if(this.state.activePillar === 'pair-button') {
+      return <Pair key="pair"/>;
+    } else if (this.state.activePillar === 'grow-button') {
+      return <Grow key="grow"/>;
+    } else if (this.state.activePillar === 'learn-button') {
+      return <Learn key="learn"/>;
+    }
+  }
+
+
+  //button shift animation
+  handleSelection = (e) => {
+    const allButtons = document.querySelectorAll('.page-button');
```

Comment:
Instead of doing this, it would be better to grab the `ref` from render and then store that in your class state

Reply:
I'm not sure I follow on this one. I'll try to dig into it some more but may need some assistance.